lib/TBDGen/CMakeLists.txt

```diff
@@ -1,10 +1,9 @@
-add_subdirectory(tapi)
-
 add_swift_host_library(swiftTBDGen STATIC
   ${TAPI_SOURCES}
```

Comment:
${TAPI_SOURCES} is no longer needed or set. I think it should be safe to remove that now too.